netsim/extra/ebgp.multihop/plugin.py

```diff
@@ -201,6 +201,13 @@ def fix_vrf_loopbacks(ndata: Box, topology: Box) -> None:
     else:
       ngb._source_ifname = ndata.loopback.ifname
 
+'''
+fix_localas_ibgp_neighbors - remove 'multihop' from any localas_ibgp neighbors
+'''
+def fix_localas_ibgp_neighbors(ndata: Box, topology: Box) -> None:
```

Comment:
I could not find where these changes are justified. Please remove them from this PR.

Reply:
replaced by separate PR